doc/src/manual/methods.md

```diff
@@ -189,7 +189,7 @@ Whoa there, Nelly.
 ```
 
 This catch-all is less specific than any other possible method definition for a pair of parameter
-values, so it is only be called on pairs of arguments to which no other method definition applies.
+values, so it is only to be called on pairs of arguments to which no other method definition applies.
```

Comment:
I'm confused, I don't see that the wording added here makes it sound like there's a choice?